docker/centos7.3/Dockerfile

```diff
@@ -0,0 +1,148 @@
+FROM docker.io/centos:7.3.1611
+MAINTAINER Hortonworks
```

Comment:
[MAINTAINER](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated) is deprecated use LABEL instead.